src/components/base/outline-alert/outline-alert.ts

```diff
@@ -0,0 +1,78 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import componentStyles from './outline-alert.css.lit';
+import { OutlineElement } from '../outline-element/outline-element';
+
+export const alertSizes = ['small', 'large'] as const;
+export type AlertSize = typeof alertSizes[number];
+
+export const alertStatusTypes = [
+  'information',
+  'warning',
+  'error',
+  'success',
+] as const;
+export type AlertStatusType = typeof alertStatusTypes[number];
+
+// This can be useful for testing.
+export interface OutlineAlertInterface extends HTMLElement {
+  statusType: AlertStatusType;
+  size: AlertSize;
+  isInteractive: Boolean;
+  shouldShowIcon: Boolean;
+}
+
+/**
+ * The Outline Alert component
+ *
+ * @element outline-alert
+ * @slot default - The alert contents
+ * @slot outline-alert--header - The header in the alert
+ */
+@customElement('outline-alert')
+export class OutlineAlert
+  extends OutlineElement
+  implements OutlineAlertInterface
+{
+  static styles: CSSResultGroup = [componentStyles];
+
+  @property({ type: String })
+  statusType: AlertStatusType = 'information';
+
+  /**
+   * This is important context for screen readers.
+   */
+  @property({ type: Boolean })
+  isInteractive = false;
+
+  @property({ type: Boolean })
```

Comment:
Even if they're simple/obvious let's keep the standard of docs for every/state/method.

Reply:
Yeah I agree, I've been doing this and reflecting that comment in the story controls. I'd like to look into creating a method for using those JSDocs to populate the storybook control info for us. Stencil does something similar.

I think for Storybook, especially, it's important to be explicit with the comments because there are QA, design, clients, etc. who are going to be seeing these pages. Its more user friendly and accessible is my argument. 

A Storybook style guide is something I'd love to get nailed down soon, I'm just trying to figure out the best way to write them still too. Would be a good discussion for guild.